pyproject.toml

```diff
@@ -15,12 +15,12 @@ authors = [
 ]
 keywords = ["MQT",  "quantum computing", "benchmarking", "performance", "testing"]
 license = { file = "LICENSE" }
-requires-python = ">=3.8"
+requires-python = ">=3.9"
 dynamic = ["version"]
 
 dependencies = [
     "qiskit[all]>=0.39.2,<0.40.0",
-    "pytket>=1.9,<1.11",
+    "pytket>=1.11,<1.12",
```

Comment:
I'd personally rather keep this change in the dependabot PR. To separate concerns a little.

Reply:
Done.